src/mrsimulator/method/tests/test_event.py

```diff
@@ -4,26 +4,27 @@
 from mrsimulator.method.event import ConstantDurationEvent
 from mrsimulator.method.event import MixingEvent
 from mrsimulator.method.event import SpectralEvent
-from mrsimulator.method.frequency_contrib import freq_default
+from mrsimulator.method.frequency_contrib import FREQ_LIST_ALL
 from mrsimulator.method.query import MixingEnum
 from pydantic import ValidationError
 
 __author__ = "Deepansh J. Srivastava"
 __email__ = "srivastava.89@osu.edu"
 
 
+# TODO: Update these test to reflect sets, or ensure an ordering in freq_contrib attr
 def test_freq_contrib():
     event = BaseEvent(freq_contrib=["Quad2_4", "Quad2_0"])
     assert event.json()["freq_contrib"] == ["Quad2_4", "Quad2_0"]
     assert event.dict()["freq_contrib"] == ["Quad2_4", "Quad2_0"]
-    assert event.json(units=False)["freq_contrib"] == ["Quad2_4", "Quad2_0"]
+    assert event.json(units=False)["freq_contrib"] == ["Quad2_0", "Quad2_4"]
```

Comment:
The problem with set is that it is unordered. These tests may or may not pass depending on the state of the set.

Do a set comparison instead.
```py
set(event.json(units=False)["freq_contrib"]) == set(["Quad2_0", "Quad2_4"])
```